libs/au-pair-cost/feature/src/lib/au-pair-cost-feature.module.ts

```diff
@@ -3,10 +3,11 @@ import { CommonModule } from '@angular/common';
 import { AuPairCostComponent } from './au-pair-cost.component';
 import { AuPairCostRoutingModule } from './au-pair-cost-routing.module';
 import { IonicModule } from '@ionic/angular';
+import { NavbarModule } from '@the-au-pair/shared/components/navbar';
 import { API } from '../../../../shared/api/api.service';
 
 @NgModule({
-  imports: [CommonModule ,AuPairCostRoutingModule, IonicModule],
+  imports: [CommonModule ,AuPairCostRoutingModule, IonicModule,NavbarModule],
```

Comment:
Need space by NavbarModule for consistency

Reply:
fixed